### PR TITLE
feature(ui-ux): added dvm evm designs for onboarding

### PR DESCRIPTION
**Pull request by nattadex**
#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode\*
- [ ] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*
- [ ] Added translations\*

### CODE REVIEW

File: mobile-app/app/screens/WalletNavigator/screens/components/OnboardingCarousel.tsx [hunk 1]
```diff
@@ -53,6 +55,13 @@ const slides: JSX.Element[] = [
...
     title="Decentralized loans"
   />,
+  <ImageSlide
+    imageDark={ImageEDark}
+    imageLight={ImageELight}
+    key={4}
+    subtitle="Seamless transfers between DeFiChain and MetaChain via generated EVM address."
+    title="EVM compatible"
+  />,
 ];
```
Comment (lykalabrada) [inline mobile-app/app/screens/WalletNavigator/screens/components/OnboardingCarousel.tsx:65]: Right side of the image is a little bit cropped
image

**Review by lykalabrada — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/WalletNavigator/screens/CreateWallet/CreateWalletGuidelines.tsx [hunk 2]
```diff
@@ -80,13 +85,21 @@ export function CreateWalletGuidelines({ navigation }: Props): JSX.Element {
...
               )}
             >
-              <ThemedIcon
-                light={tailwind("text-mono-light-v2-100")}
-                dark={tailwind("text-mono-dark-v2-100")}
-                iconType="Feather"
-                name={g.icon}
-                size={14}
-              />
+              {g.icon ? (
```
Comment (fullstackninja864) [inline mobile-app/app/screens/WalletNavigator/screens/CreateWallet/CreateWalletGuidelines.tsx:88]: For consistency can we do ?

```
interface GuidelineItem {
  title: string;
  Icon?: () => JSX.Element
}

File: shared/translations/languages/zh-Hant.json [hunk 3]
```diff
@@ -610,6 +610,8 @@
     "Trade on the DEX and earn rewards from liquidity mining with crypto and dTokens.": "在 DEX 上進行交易並使用加密貨幣和 dToken 從流動性挖礦中獲得獎勵 ",
     "Decentralized loans": "去中心化資產貸款",
     "Access financial opportunities with dTokens minted through decentralized vaults.": "透過去中心化金庫鑄造的 dToken 獲得交易機會",
+    "EVM compatible":"EVM compatible",
```
Comment (fullstackninja864) [inline shared/translations/languages/zh-Hant.json:613]:
```suggestion
    "EVM compatible": "EVM compatible",
```

File: shared/translations/languages/zh-Hant.json [hunk 4]
```diff
@@ -610,6 +610,8 @@
...
+    "Seamless transfers between DeFiChain and MetaChain via generated EVM address.":"Seamless transfers between DeFiChain and MetaChain via generated EVM address.",
```
Comment (fullstackninja864) [inline shared/translations/languages/zh-Hant.json:614]:
```suggestion
    "Seamless transfers between DeFiChain and MetaChain via generated EVM address.": "Seamless transfers between DeFiChain and MetaChain via generated EVM address.",
```

File: shared/translations/languages/zh-Hant.json [hunk 5]
```diff
@@ -643,7 +645,8 @@
...
     "Write the words on paper. Take note of their correct spelling and order.": "Write the words on paper. Take note of their correct spelling and order.",
-    "Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone.": "Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone."
+    "Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone.": "Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone.",
+    "A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain.":"A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain."
```
Comment (fullstackninja864) [inline shared/translations/languages/zh-Hant.json:649]:
```suggestion
    "A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain.": "A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain."
```

File: shared/translations/languages/zh-Hans.json [hunk 6]
```diff
@@ -643,7 +645,8 @@
...
     "Write the words on paper. Take note of their correct spelling and order.": "Write the words on paper. Take note of their correct spelling and order.",
-    "Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone.": "Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone."
+    "Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone.": "Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone.",
+    "A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain.":"A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain."
```
Comment (fullstackninja864) [inline shared/translations/languages/zh-Hans.json:649]:
```suggestion
    "A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain.": "A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain."
```

File: shared/translations/languages/zh-Hans.json [hunk 7]
```diff
@@ -610,6 +610,8 @@
     "Trade on the DEX and earn rewards from liquidity mining with crypto and dTokens.": "在 DEX 上进行交易并使用加密货币和 dToken 从流动性挖矿中获得奖励 ",
     "Decentralized loans": "去中心化资产贷款",
     "Access financial opportunities with dTokens minted through decentralized vaults.": "透过去中心化金库铸造的 dToken 获得交易机会",
+    "EVM compatible":"EVM compatible",
+    "Seamless transfers between DeFiChain and MetaChain via generated EVM address.":"Seamless transfers between DeFiChain and MetaChain via generated EVM address.",
```
Comment (fullstackninja864) [inline shared/translations/languages/zh-Hans.json:614]:
```suggestion
    "Seamless transfers between DeFiChain and MetaChain via generated EVM address.": "Seamless transfers between DeFiChain and MetaChain via generated EVM address.",
```

File: shared/translations/languages/zh-Hans.json [hunk 8]
```diff
@@ -610,6 +610,8 @@
     "Trade on the DEX and earn rewards from liquidity mining with crypto and dTokens.": "在 DEX 上进行交易并使用加密货币和 dToken 从流动性挖矿中获得奖励 ",
     "Decentralized loans": "去中心化资产贷款",
     "Access financial opportunities with dTokens minted through decentralized vaults.": "透过去中心化金库铸造的 dToken 获得交易机会",
+    "EVM compatible":"EVM compatible",
```
Comment (fullstackninja864) [inline shared/translations/languages/zh-Hans.json:613]:
```suggestion
    "EVM compatible": "EVM compatible",
```

File: shared/translations/languages/it.json [hunk 9]
```diff
@@ -659,7 +661,8 @@
...
     "Write the words on paper. Take note of their correct spelling and order.": "Write the words on paper. Take note of their correct spelling and order.",
-    "Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone.": "Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone."
+    "Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone.": "Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone.",
+    "A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain.":"A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain."
```
Comment (fullstackninja864) [inline shared/translations/languages/it.json:665]:
```suggestion
    "A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain.": "A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain."
```

File: shared/translations/languages/it.json [hunk 10]
```diff
@@ -624,6 +624,8 @@
     "Trade on the DEX and earn rewards from liquidity mining with crypto and dTokens.": "Scambia i token per ottenere rendimenti elevati e guadagna i premi di liquidity mining con i token decentralizzati.",
     "Decentralized loans": "Prestiti decentralizzati",
     "Access financial opportunities with dTokens minted through decentralized vaults.": "Conia e accedi a nuovi dToken utilizzando caveau sovra-collateralizzati.",
+    "EVM compatible":"EVM compatible",
+    "Seamless transfers between DeFiChain and MetaChain via generated EVM address.":"Seamless transfers between DeFiChain and MetaChain via generated EVM address.",
```
Comment (fullstackninja864) [inline shared/translations/languages/it.json:628]:
```suggestion
    "EVM compatible": "EVM compatible",
    "Seamless transfers between DeFiChain and MetaChain via generated EVM address.": "Seamless transfers between DeFiChain and MetaChain via generated EVM address.",
```

File: shared/translations/languages/fr.json [hunk 11]
```diff
@@ -651,6 +653,7 @@
     "I understand it is my responsibility to keep my recovery words secure. Losing them will result in the irrecoverable loss of access to my wallet funds.": "Je comprends qu'il est de ma responsabilité de garder mes mots de récupération en sécurité. Leur perte entraînera la perte irrémédiable de l'accès aux fonds de mon portefeuille.",
     "Write the words on paper. Take note of their correct spelling and order.": "Prenez note des mots sur papier. Faites bien attention à l'orthographe et à respecter l'ordre.",
     "Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone.": "Sécurisez-les dans un endroit sûr. Conservez-les hors ligne et dans un endroit sûr auquel vous êtes le seul à avoir accès. Gardez-les secrets et ne partagez pas avec qui que ce soit.",
+    "A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain.":"Une adresse EVM mappée sera générée pour assurer des transferts transparents entre DeFiChain et MetaChain.",
```
Comment (fullstackninja864) [inline shared/translations/languages/fr.json:656]:
```suggestion
    "A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain.": "Une adresse EVM mappée sera générée pour assurer des transferts transparents entre DeFiChain et MetaChain.",
```

File: shared/translations/languages/fr.json [hunk 12]
```diff
@@ -619,6 +619,8 @@
     "Trade on the DEX and earn rewards from liquidity mining with crypto and dTokens.": "Négociez sur le DEX et gagnez des récompenses de liquidity mining avec des cryptomonnaies et des dTokens.",
     "Decentralized loans": "Prêts décentralisés",
     "Access financial opportunities with dTokens minted through decentralized vaults.": "Accédez à des opportunités financières avec des dTokens mintés par des vaults décentralisés.",
+    "EVM compatible":"Compatible EVM",
+    "Seamless transfers between DeFiChain and MetaChain via generated EVM address.":"Transferts transparents entre DeFiChain et MetaChain via l'adresse EVM générée.",
```
Comment (fullstackninja864) [inline shared/translations/languages/fr.json:623]:
```suggestion
    "EVM compatible": "Compatible EVM",
    "Seamless transfers between DeFiChain and MetaChain via generated EVM address.": "Transferts transparents entre DeFiChain et MetaChain via l'adresse EVM générée.",
```

File: shared/translations/languages/es.json [hunk 13]
```diff
@@ -660,7 +662,8 @@
...
     "Write the words on paper. Take note of their correct spelling and order.": "Escribe las palabras en papel. Manten el orden y la ortografía correctos.",
-    "Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone.": "Guardalas en un lugar seguro. Mantenlas offline en un sitio al que solo tu tengas acceso. Mantenlas privadas y no las compartas con nadie."
+    "Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone.": "Guardalas en un lugar seguro. Mantenlas offline en un sitio al que solo tu tengas acceso. Mantenlas privadas y no las compartas con nadie.",
+    "A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain.":"A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain."
```
Comment (fullstackninja864) [inline shared/translations/languages/es.json:666]:
```suggestion
    "A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain.": "A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain."
```

File: shared/translations/languages/es.json [hunk 14]
```diff
@@ -625,6 +625,8 @@
     "Trade on the DEX and earn rewards from liquidity mining with crypto and dTokens.": "Intercambia tokens para obtener recompensas del liquidity mining con los tokens descentralizados.",
     "Decentralized loans": "Préstamos descentralizados",
     "Access financial opportunities with dTokens minted through decentralized vaults.": "Crea y accede a nuevos dTokens usando Vaults sobre-colateralizados",
+    "EVM compatible":"EVM compatible",
+    "Seamless transfers between DeFiChain and MetaChain via generated EVM address.":"Seamless transfers between DeFiChain and MetaChain via generated EVM address.",
```
Comment (fullstackninja864) [inline shared/translations/languages/es.json:629]:
```suggestion
    "EVM compatible": "EVM compatible",
    "Seamless transfers between DeFiChain and MetaChain via generated EVM address.": "Seamless transfers between DeFiChain and MetaChain via generated EVM address.",
```

File: shared/translations/languages/de.json [hunk 15]
```diff
@@ -641,7 +643,8 @@
...
     "Write the words on paper. Take note of their correct spelling and order.": "Schreibe die Wörter auf Papier. Achte auf die richtige Schreibweise und Reihenfolge.",
-    "Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone.": "Bewahre sie an einem sicheren Ort auf. Speichere sie offline an einem Ort, zu dem nur du Zugang hast. Schütze sie vor fremden Einblicken und teile sie mit niemandem."
+    "Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone.": "Bewahre sie an einem sicheren Ort auf. Speichere sie offline an einem Ort, zu dem nur du Zugang hast. Schütze sie vor fremden Einblicken und teile sie mit niemandem.",
+    "A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain.":"Eine zugeordnete EVM-Adresse wird generiert, um eine nahtlose Übertragung zwischen DeFiChain und MetaChain zu gewährleisten."
```
Comment (fullstackninja864) [inline shared/translations/languages/de.json:647]:
```suggestion
    "A mapped EVM address will be generated to ensure seamless transfers between DeFiChain and MetaChain.": "Eine zugeordnete EVM-Adresse wird generiert, um eine nahtlose Übertragung zwischen DeFiChain und MetaChain zu gewährleisten."
```

File: shared/translations/languages/de.json [hunk 16]
```diff
@@ -609,6 +609,8 @@
     "Trade on the DEX and earn rewards from liquidity mining with crypto and dTokens.": "Handle auf der DEX und verdiene Rewards aus dem Liquiditäts-Mining mit Kryptos und dToken.",
     "Decentralized loans": "Dezentrale Darlehen",
     "Access financial opportunities with dTokens minted through decentralized vaults.": "Erhalte Zugang zu finanziellen Gelegenheiten mit dToken, die in dezentralen Vaults gemintet wurden.",
+    "EVM compatible":"EVM-kompatibel",
+    "Seamless transfers between DeFiChain and MetaChain via generated EVM address.":"Nahtlose Übertragung zwischen DeFiChain und MetaChain durch generierte EVM-Adresse.",
```
Comment (fullstackninja864) [inline shared/translations/languages/de.json:613]:
```suggestion
    "EVM compatible": "EVM-kompatibel",
    "Seamless transfers between DeFiChain and MetaChain via generated EVM address.": "Nahtlose Übertragung zwischen DeFiChain und MetaChain durch generierte EVM-Adresse.",
```

**Review by fullstackninja864 — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/WalletNavigator/screens/CreateWallet/CreateWalletGuidelines.tsx [hunk 17]
```diff
@@ -18,26 +17,59 @@ import DarkNewWallet from "@assets/images/dark-wallet-guidelines.png";
...
+      return (
+        <ThemedIcon
+          light={tailwind("text-mono-light-v2-100")}
+          dark={tailwind("text-mono-dark-v2-100")}
+          iconType="Feather"
+          name="edit-2"
+          size={14}
+        />
+      );
+    },
```
Comment (fullstackninja864) [inline mobile-app/app/screens/WalletNavigator/screens/CreateWallet/CreateWalletGuidelines.tsx:45]:
```suggestion
    Icon: (): JSX.Element => (
        <ThemedIcon
          light={tailwind("text-mono-light-v2-100")}
          dark={tailwind("text-mono-dark-v2-100")}
          iconType="Feather"
          name="edit-2"
          size={14}
        />
      ),
```

**Review by fullstackninja864 — APPROVED** [review]

**Review by lykalabrada — APPROVED** [review]

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**